### PR TITLE
Minor Boiler QOL changes

### PR DESCRIPTION
**Pull request by Notamaniac**
## About The Pull Request
Changes the pips on globule storage to simple numbers for readability purposes
globstorage
Removes the dedicated 60(!!) sprite icon file for globules
Changes globule creation tochat() messages to balloon alerts, with the amount of globlues currently in reserve.

## Why It's Good For The Game

Mostly for readability, in the midst of combat you don't really have that much time to squint and count how many pips you have.

## Changelog
:cl:
qol: Boiler globules in reserve are now listed as a number on the create globule ability button. You're also told how many globules of your current acid type are in reserve when making a globule.
qol: Changed several boiler to_chat() messages to balloon alerts.
/:cl:

### CODE REVIEW

**Comment by dopamiin0** [comment]
i didnt even know there was a pip change i just prayed or tried to count in my head this is huge

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm [hunk 1]
```diff
@@ -147,27 +159,34 @@ GLOBAL_LIST_INIT(boiler_glob_image_list, list(
...
 
 	succeed_activate()
 	if(istype(X.ammo, /datum/ammo/xeno/boiler_gas/corrosive))
 		X.corrosive_ammo++
-		to_chat(X, span_notice("We prepare a corrosive acid globule."))
+		X.balloon_alert(X,"Acid globule prepared - [X.corrosive_ammo] in reserve.")
 	else
 		X.neuro_ammo++
-		to_chat(X, span_notice("We prepare a neurotoxic gas globule."))
+		X.balloon_alert(X,"Neuro globule prepared - [X.neuro_ammo] in reserve.")
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm:176]: As a rule try keep balloon alerts very short. They have key information from A: they presumably know what buttons they're pressing, and B: The action button has more info that they can read.

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm [hunk 2]
```diff
@@ -130,14 +130,26 @@ GLOBAL_LIST_INIT(boiler_glob_image_list, list(
...
+	var/mutable_appearance/neuroglob_maptext = mutable_appearance(icon = null, icon_state = null, layer = ACTION_LAYER_MAPTEXT)
+	visual_references[VREF_MUTABLE_NEUROGLOB_COUNTER] = neuroglob_maptext
+	neuroglob_maptext.pixel_x = 25
+	neuroglob_maptext.pixel_y = -4
+	neuroglob_maptext.maptext = "<span class='maptext' style=font-size:8px><font color=yellow>[X.neuro_ammo]</span>"
+	var/mutable_appearance/corrosiveglob_maptext = mutable_appearance(icon = null, icon_state = null, layer = ACTION_LAYER_MAPTEXT)
+	visual_references[VREF_MUTABLE_CORROSIVEGLOB_COUNTER] = corrosiveglob_maptext
+	corrosiveglob_maptext.pixel_x = 25
+	corrosiveglob_maptext.pixel_y = 8
+	corrosiveglob_maptext.maptext = "<span class='maptext' style=font-size:8px><font color=green>[X.corrosive_ammo]</span>"
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm:152]: span macros

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm [hunk 3]
```diff
@@ -130,14 +130,26 @@ GLOBAL_LIST_INIT(boiler_glob_image_list, list(
...
 		KEYBINDING_NORMAL = COMSIG_XENOABILITY_CREATE_BOMB,
 	)
+/datum/action/ability/xeno_action/create_boiler_bomb/give_action(mob/living/L)
+	. = ..()
+	var/mob/living/carbon/xenomorph/boiler/X = owner
+	var/mutable_appearance/neuroglob_maptext = mutable_appearance(icon = null, icon_state = null, layer = ACTION_LAYER_MAPTEXT)
+	visual_references[VREF_MUTABLE_NEUROGLOB_COUNTER] = neuroglob_maptext
+	neuroglob_maptext.pixel_x = 25
+	neuroglob_maptext.pixel_y = -4
+	neuroglob_maptext.maptext = "<span class='maptext' style=font-size:8px><font color=yellow>[X.neuro_ammo]</span>"
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm:147]: span macros

File: code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm [hunk 4]
```diff
@@ -147,27 +159,34 @@ GLOBAL_LIST_INIT(boiler_glob_image_list, list(
...
 /datum/action/ability/xeno_action/create_boiler_bomb/update_button_icon()
 	var/mob/living/carbon/xenomorph/boiler/X = owner
-	action_icon_state = "bomb_count_[X.corrosive_ammo][X.neuro_ammo]"
+	button.cut_overlay(visual_references[VREF_MUTABLE_CORROSIVEGLOB_COUNTER])
+	var/mutable_appearance/corrosiveglobnumber = visual_references[VREF_MUTABLE_CORROSIVEGLOB_COUNTER]
+	corrosiveglobnumber.maptext = "<span class='maptext' style=font-size:8px><font color=green>[X.corrosive_ammo]</span>"
+	button.add_overlay(visual_references[VREF_MUTABLE_CORROSIVEGLOB_COUNTER])
+	button.cut_overlay(visual_references[VREF_MUTABLE_NEUROGLOB_COUNTER])
+	var/mutable_appearance/neuroglobnumber = visual_references[VREF_MUTABLE_NEUROGLOB_COUNTER]
+	neuroglobnumber.maptext = "<span class='maptext' style=font-size:8px><font color=yellow>[X.neuro_ammo]</span>"
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm:188]: some more span macros

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Comment by Lumipharon** [comment]
Don't self close reviews please, it makes it hard to check

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**